BUILDING_ON_LINUX.md

```diff
@@ -34,50 +35,48 @@ _most likely works the same for other Debian-like distros_
 
 ### Manually
 
-1. `sudo pacman -S qt5-base qt5-multimedia qt5-svg qt5-tools gst-plugins-ugly gst-plugins-good boost rapidjson pkgconf openssl cmake`
-1. go into project directory
-1. create build folder `mkdir build && cd build`
+1. Install all of the dependencies using `sudo pacman -S qt5-base qt5-multimedia qt5-svg qt5-tools gst-plugins-ugly gst-plugins-good boost rapidjson pkgconf openssl cmake`
+1. Go into the project directory
+1. Create a build folder and open it (`mkdir build && cd build`)
+1. Use one of the options below to compile it
 
-#### Using QMake
+### Using CMake
 
-1. `qmake .. && make`
+`cmake .. && make`
 
-#### Using CMake
+### Using QMake
 
-1. `cmake .. && make`
+`qmake .. && make`
 
 ## Fedora 28 and above
 
-_most likely works the same for other Red Hat-like distros. Substitue `dnf` with `yum`._
-
-1. `sudo dnf install qt5-qtbase-devel qt5-qtmultimedia-devel qt5-qtsvg-devel libsecret-devel openssl-devel boost-devel cmake`
-1. go into project directory
-1. create build folder `mkdir build && cd build`
-
-### Using QMake
+_Most likely works the same for other Red Hat-like distros. Substitute `dnf` with `yum`._
 
-1. `qmake-qt5 .. && make -j$(nproc)`
+1. Install all of the dependencies using `sudo dnf install qt5-qtbase-devel qt5-qtmultimedia-devel qt5-qtsvg-devel libsecret-devel openssl-devel boost-devel cmake`
+1. Go into the project directory
+1. Create a build folder and open it (`mkdir build && cd build`)
+1. Use one of the options below to compile it
 
 ### Using CMake
 
-1. `cmake .. && make -j$(nproc)`
+`cmake .. && make -j$(nproc)`
 
-### Optional dependencies
-
-_`gstreamer-plugins-good` package is retired in Fedora 31, see: [rhbz#1735324](https://bugzilla.redhat.com/show_bug.cgi?id=1735324)_
+### Using QMake
 
-1. `sudo dnf install gstreamer-plugins-good` _(optional: for audio output)_
+`qmake-qt5 .. && make -j$(nproc)`
 
 ## NixOS 18.09+
 
-1. enter the development environment with all of the dependencies: `nix-shell -p openssl boost qt5.full pkg-config cmake`
-1. go into project directory
-1. create build folder `mkdir build && cd build`
+1. Enter the development environment with all of the dependencies: `nix-shell -p openssl boost qt5.full pkg-config cmake`
+1. Go into the project directory
+1. Create a build folder and open it (`mkdir build && cd build`)
+1. Use one of the options below to compile it
 
-### Using QMake
+### Using CMake
 
-1. `qmake .. && make`
+`cmake .. && make`
 
-### Using CMake
 
-1. `cmake .. && make`
+### Using QMake
+
+`qmake .. && make`
```

Comment:
Why was the order reversed? QMake is still the first building system that is used and CMake has been only added later. Imho this should stay as it and then maybe we'd just remove QMake altogether once we drop it.

Reply:
IMO it doesn't really matter that much, while there is no reason for it to be this way now, there really isn't a reason for reverting in back to how it was before